test/test_account_info.py

```diff
@@ -58,85 +60,38 @@ def test_clear(self):
         self.assertEqual((None, None), self.pool.take('b'))
 
 
-class TestSqliteAccountInfo(TestBase):
-    def __init__(self, *args, **kwargs):
-        super(TestSqliteAccountInfo, self).__init__(*args, **kwargs)
-        self.db_path = tempfile.NamedTemporaryFile(
-            prefix='tmp_b2_tests_%s__' % (self.id(),), delete=True
-        ).name
-
-    def setUp(self):
-        if platform.system().lower().startswith('java'):
-            # in Jython 2.7.1b3 there is no sqlite3
-            raise SkipTest()
-        try:
-            os.unlink(self.db_path)
-        except OSError:
-            pass
-        print('using %s' % self.db_path)
-
-    def tearDown(self):
-        try:
-            os.unlink(self.db_path)
-        except OSError:
-            pass
-
-    def test_account_info(self):
-        account_info = self._make_info()
-        account_info.set_auth_data(
-            'account_id', 'account_auth', 'api_url', 'download_url', 100, 'app_key', 'realm'
-        )
+@six.add_metaclass(ABCMeta)
+class AccountInfoBase(object):
```

Comment:
Alternative is to set a `__test__` field to `False`, but that gets inherited, so all child classes need to override this field or they will not get their tests called. I feared this will be error prone.